app/test/frontend/golden/pkg_index_page.html

```diff
@@ -16,6 +16,7 @@
     <meta itemprop="image" content="/static/img/dart-logo-400x400.png?hash=mocked_hash_58383222"/>
     <meta property="og:image" content="/static/img/dart-logo-400x400.png?hash=mocked_hash_58383222"/>
     <link rel="shortcut icon" href="/favicon.ico?hash=mocked_hash_985685822"/>
+    <link rel="search" type="application/opensearchdescription+xml" title="Dart packages" href="/static/osd.xml?hash=mocked_hash_869379763">
```

Comment:
All of the tests require a closing tag (you've updated like the half of it).
If you want to, I can take over from here, it is not that fun to update these tests by hand.

Reply:
Perhaps, my attempts at using find/replace are being pretty agonizingly slow.